src/components/routes/homepage/MediaCard.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import moment from 'moment';
+
+import {
+  Card,
+  CardActionArea,
+  CardMedia,
+  CardContent,
+  Typography,
+  makeStyles,
+  ExpansionPanel,
+  ExpansionPanelSummary,
+  ExpansionPanelDetails
+} from '@material-ui/core';
+
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+
+const useStyles = makeStyles({
+  root: {
+    maxWidth: 345,
+    padding: '10px',
+    fontFamily: "'Merriweather', serif"
+  },
+  media: {
+    height: 140,
+    fontFamily: "'Merriweather', serif"
+  },
+  expansionPanel: {
+    lineHeight: '0px',
+    marginBottom: '0px',
+    fontFamily: "'Merriweather', serif"
+  }
+});
+
+export const MediaCard = ({ event }) => {
+  //TODO: add real image instead of using boilerplate image (./yiruma.jpg)
+  //      we have to wait for phoenix to be properly set up (base64 -> url convertion and store in S3)
+  const classes = useStyles();
+  const {
+    title,
+    start,
+    end,
+    venue,
+    description,
+    organiser: { username, email }
+  } = event;
+
+  return (
+    <Card className={classes.root}>
+      <CardActionArea>
+        <CardMedia
+          style={{ height: 0, paddingTop: '56%' }}
+          className={classes.media}
+          image={require('./yiruma.jpg')}
+          title="Yiruma: Pianist"
```

Comment:
Consider moving the styles out of here.
Also, the image seems to be statically imported. Note that this will change just like the title etc.

Reply:
yup, I used a static image url, will change it

Reply:
@utkarsh867 I have one quick question about this actually. Should phoenix send back `image_url` or `image_base64 ` to raven??

Reply:
Let's take this decision to the meeting.